src/main/java/frc/robot/subsystems/arm/PrototypeArm.java

```diff
@@ -0,0 +1,145 @@
+package frc.robot.subsystems.arm;
+
+import com.ctre.phoenix.motorcontrol.*;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.ctre.phoenix.sensors.CANCoder;
+import edu.wpi.first.math.geometry.Translation2d;
+import frc.robot.Ports;
+import frc.robot.subsystems.LoggedSubsystem;
+import frc.robot.utils.units.UnitModel;
+
+public class PrototypeArm extends LoggedSubsystem<PrototypeArmLogInputs> {
+    private final ArmKinematics kinematics = new ArmKinematics(ArmConstants.SHOULDER_ARM_LENGTH, ArmConstants.ELBOW_ARM_LENGTH);
+    private final ArmSystemModel systemModel = new ArmSystemModel(ArmConstants.ARM_CONSTANTS);
+
+    public final edu.wpi.first.wpilibj.Timer timer = new edu.wpi.first.wpilibj.Timer();
```

Comment:
Fix the imports so that line will look line:
```suggestion
    public final Timer timer = new Timer();
```